site/ar/tutorials/images/data_augmentation.ipynb

```diff
@@ -0,0 +1,911 @@
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "قم بإشباع الصورة من خلال توفير عامل إشباع بالطريقة التّالية:\n",
+        "\n",
+        "</div>"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "tiTUhw-gzCJW"
+      },
+      "outputs": [],
+      "source": [
+        "saturated = tf.image.adjust_saturation(image, 3)\n",
+        "visualize(image, saturated)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "E82CqomP9qcR"
+      },
+      "source": [
+        "### <div dir=\"rtl\">تغيير درجة سطوع الصورة</div>\n",
+        "\n",
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "قم بتغيير درجة سطوع الصورة عن طريق توفير عامل سطوع بالطريقة التّالية:\n",
+        "\n",
+        "</div>"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "05dA6uEtzfyd"
+      },
+      "outputs": [],
+      "source": [
+        "bright = tf.image.adjust_brightness(image, 0.4)\n",
+        "visualize(image, bright)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "5_0kMbmS91x6"
+      },
+      "source": [
+        "### <div dir=\"rtl\">تدوير الصورة</div>\n",
+        "\n",
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "قم بتدوير الصورة 90 درجة للحصول على صورة أخرى بالطريقة التّالية:\n",
+        "\n",
+        "</div>"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "edNoQzhszxo8"
+      },
+      "outputs": [],
+      "source": [
+        "rotated = tf.image.rot90(image)\n",
+        "visualize(image, rotated)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "bomBnFWp9895"
+      },
+      "source": [
+        "### <div dir=\"rtl\">اقتصاص الصورة في المنتصف</div>\n",
+        "\n",
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "قم باقتصاص الصورة في المنتصف إلى الحدّ الذّي تريده بالطريقة التّالية:\n",
+        "\n",
+        "</div>"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "fvgz_6t21dq2"
+      },
+      "outputs": [],
+      "source": [
+        "cropped = tf.image.central_crop(image, central_fraction=0.5)\n",
+        "visualize(image,cropped)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "8W5E_c7o-H96"
+      },
+      "source": [
+        "<div dir=\"rtl\">\n",
+        "\n",
+        "أنظر في تفاصيل دليل الدّالة \n",
+        "`tf.image`\n",
```

Comment:
Add the link to tf.image like the original version

Reply:
As you can see in the english version, there are no links in the notebook: https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/images/data_augmentation.ipynb

However, the pipeline that generates the website adds the links automatically, that's why they show in the website.